examples/acados_matlab_octave/pendulum_on_cart_model/minimal_example_ocp.m

```diff
@@ -41,11 +41,11 @@
 end
 
 %% discretization
-N = 100;
+N = 20;
```

Comment:
Do you still see the swing up with 20 stages?

Reply:
yes, the plot doesnt look that smooth though.
I just used it to check the templating stuff with less output.

Reply:
Indeed, this is what is implemented in the SCQP PR ;)